packages/server/lib/environment.coffee

```diff
@@ -39,6 +40,14 @@ try
   if os.platform() is "linux"
     app.disableHardwareAcceleration()
 
+  if process.env.ELECTRON_EXTRA_LAUNCH_ARGS
+    electronLaunchArgs = parseElectronLaunchArguments(process.env.ELECTRON_EXTRA_LAUNCH_ARGS)
+    Object.keys(electronLaunchArgs).forEach ({key, value}) ->
+      if value == undefined
+        app.commandLine.appendSwitch(key)
+      else
+        app.commandLine.appendSwitch(key, value)
+
```

Comment:
I think most of this complexity could go away if you used `app.commandLine.appendArgument`, you wouldn't need to split `key` and `value`, you could just pass it directly to `appendArgument` as one complete arg.